carball/analysis/analysis_manager.py

```diff
@@ -31,6 +31,17 @@
 
 
 class AnalysisManager:
+    """
+    DESCRIPTION
+    AnalysisManager class takes an initialized Game object and converts the data into a Protobuf and a DataFrame. Then,
+    that data is used to perform full analysis on the replay.
+
+    COMMON PARAMS
+    :game: The game object (instance of Game). It contains the replay metadata and processed json data.
+    :proto_game: The game's protobuf (instance of game_pb2) (refer to the comment in get_protobuf_data() for more info).
+    :data_frame: The game's pandas.DataFrame object (refer to comment in get_data_frame() for more info).
```

Comment:
Maybe we should copy the example from the GitHub on how to create an instance and use it.

Also "params" don't make sense because they are not passed into the `__init__` method

Reply:
Sure, I just thought it's easier to state it at the beginning and refer to it. Will change